AirCasting/Utils/DataFormatters.swift

```diff
@@ -0,0 +1,100 @@
+// Created by Lunar on 30/09/2021.
+//
+
+import Foundation
+
+enum DateFormatters {
+    
+    enum SessionDownloadService {
+        static let decoderDateFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.timeZone = TimeZone.init(abbreviation: "UTC")
+            df.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+            return df
+        }()
+    }
+    
+    enum SessionUploadService {
+        static let encoderDateFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.timeZone = TimeZone.init(abbreviation: "UTC")
+            df.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+            return df
+        }()
+    }
+    
+    enum CreateSessionAPIService {
+        static let encoderDateFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+            return df
+        }()
+    }
+    
+    enum AirBeam3Configurator {
+        static let USLocaleDateFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+            df.locale = Locale(identifier: "en_US_POSIX")
+            return df
+        }()
+    }
+    
+    enum SessionCartView {
+        static let pollutionChartDateFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.dateFormat = "HH:mm"
+            return df
+        }()
+    }
+    
+    enum GraphView {
+        static let USTimeDateFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.dateFormat = "HH:mm"
+            df.locale = Locale(identifier: "en_US")
+            return df
+        }()
+        
+        static let mobileActiveDateFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.dateFormat = "HH:mm"
+            df.timeZone = TimeZone.init(abbreviation: "UTC")
+            df.locale = Locale(identifier: "en_US")
+            return df
+        }()
+    }
+    
+    enum TimeAxisRenderer {
+        static let shortUSDateFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.timeStyle = .short
+            df.locale = Locale(identifier: "en_US")
+            return df
+        }()
+    }
+    
+    enum DateExtension {
+        static let SSSdateFormatter: DateFormatter = {
```

Comment:
still don't like the name ;) how about `miliseconds` or to conform to established standard here `milisecondsDateFormatter` (but I think that there is no need to actually put the `dateFormatter` in this names since the caller will already have to explicitly type it when accessing: `DateFormatters.xxxx.xxxx`. But that's a matter of taste, so no need to implement this)